src/components/Article.jsx

```diff
@@ -1,13 +1,7 @@
 import React from 'react';
-// import { useNavigate } from 'react-router-dom';
 import dayjs from 'dayjs';
 
 const Article = props => {
```

Comment:
does article and singleArticle do the same thing? i'm unable to see it live due to the errors of 'cannot read properties of undefined, reading map' at ArticleList.jsx:16

Reply:
Hi, it's because this is the second pull request, some of the code is on the previous one i think :) i'm not having any errors